examples/simple/manifest.json

```diff
@@ -1,30 +1,15 @@
 {
     "server": {
+        "debug": false // disable Hapi debug console logging
     },
     "register": {
         "hapi-pino": {
             "plugin": "require:hapi-pino",
-            "options": {
-                "mergeHapiLogData": false,
-                // "level": "debug",
-                "ignorePaths": [
-                    "/health",
-                    "/alive.txt",
-                    "/private"
-                ],
-                "tags": { "GET_items": "info", "GET_error": "error" },
-                // transport option is not recomended for prod env
-                "transport": {
-                    "target": "pino-pretty",
-                    "options": {
-                        "singleLine": true,
-                        "colorize": true,
-                        "mkdir": true,
-                        "append": false,
-                        "destination": "logs/output.log"
-                    }
-                }
-            }
+            "options":{
+                "logRequestComplete": false,
+                "ignoredEventTags": { "log": ["client"], "request": "*" },
+                "logPayload": false
+              }
```

Comment:
this should include a `transport` and/or a `pino-pretty` one for `development`.

Reply:
We already are passing the transport config with pino-pretty at our base file:

https://github.com/ExpediaGroup/catalyst-server/blob/master/lib/config.json#L43

But yeah is better if we pass empty there and pass the config to this example...

why?

in same cases/config pino will fail at this validation:

https://github.com/pinojs/pino/blob/6030b2670d2f0b0f106d9d8c59a5d2b1675749f3/lib/transport.js#L93